content/04.results.md

```diff
@@ -0,0 +1,16 @@
+# Results
+
+1. Figure 1 - Visual to show that alignment works along intra year variation (Procrustes Validation)
+  1. Umap panel for an individual year to show models are separated
+  2. Umap panel to show the same year after models have been aligned
+  3. Umap panel to show across years (inter-year varation)
+  	  1. might not have this as a figure given the aligned umap results, but we shall see 
+2. Figure 2 - CUSUM validation
```

Comment:
I think you might want another figure here that shows how you can use the variability in distance to address the challenges of variable training set size / model uncertainty. Let's see how things shake out as you're writing.